docs/aws/GettingStarted/nRFCloudLocationServices.rst

```diff
@@ -0,0 +1,33 @@
+.. _aws-nrf-cloud-location-services:
+
+nRF Cloud Location Services
+###########################
+
+nRF Cloud Location Services are an essential component of any asset tracking solution and nRF Asset Tracker makes heavy use of them.
```

Comment:
This change is not required since the original is not really wrong, just maybe overselling how essential nRF Cloud Location Services are.
```suggestion
Location services are an essential component of any asset tracking solution.
For nRF Asset Tracker, the service used is nRF Cloud Location Services.
```

Reply:
Yes, the boost is intentional.